docs/user-guide/team-planner/README.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_navigation:
+  title: Team planner
+  priority: 890
+description: Using the team planner in OpenProject.
+robots: index, follow
+keywords: team planner,planning calendar, resource management
+---
+
+# Team planner
+
+The team planner is a module that can assist getting a better overview of which team member is working on what work package at what time. It is also a useful tool to schedule those team members for different tasks on various times.
+
+A few notes: 
+
+1. The work package module has to be enabled as a prerequisite for the team planner
+2. You can have multiple public, private, as well as favorite team planning calendars. They will all appear in the left menu bar.
+
+| Topic                 | Content                                     |
+| ------------------------------------- | :------------------------------------------ |
+| [Main view - planning calendar basics](team-planner) | How to navigate the team planner? |
+| **Getting started** |  |
+| [Step 1: Adding team members](team-planner) | How to add team members? |
+| [Step 2: Adding/working with work package strips](team-planner) | How to work with work package strips? |
+| [Work package strip detail view](team-planner) | How to access the work package detail view? |
+| [Removing a work package strip](team-planner) | How to remove a work package strip? |
+
+## Main view - Start a new or select an existing Team Planner
+
+![Overview of the team planner](Team-planner-overview.png)
+
+
+
+1. You can create a new team planner using either the "+ Create" button in the top right near your avatar, or by clicking the "+ New team planner" button at the bottom of the black menu bar.
+
+2. If you created a team planner before, you can select one of the planners from the list.
+
+3. Team planners with the visibility option "public" selected appear under the "Public Views" heading in the black menu bar. 
+
+4. Team planners with the visibility option "public" unselected appear under the "Private Views" heading in the black menu bar. Note: If "public" is not selected, the calendar is automatically "private" and can only be seen by you, not other members.
+
+   Both "public" and "private" calendars can be added as a "favored" planner, and would appear under the heading "Favorite Views" in the black menu bar.
+
```

Comment:
https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#ui-text
instead of using inverted commas, please make the terms bold

Reply:
Thank you Rebecca! Changes submitted.